mlflow/entities/span.py

```diff
@@ -28,32 +29,28 @@ class SpanType:
 
 @dataclass
 class Span(_MlflowObject):
-    """A span object. (TODO: Add conceptual guide for span vs trace.)
+    """A span object. OpenTelemetry compatible but defines subset of fields.
 
     Args:
         name: Name of the span.
-        context: SpanContext object that contains the request_id (= trace_id) and span_id.
-        parent_span_id: Id of the parent span. If None, the span is the root span.
+        context: SpanContext object that contains the trace_id and span_id.
+        parent_id: Id of the parent span. If None, the span is the root span.
         status: Status of the span.
         start_time: Start time of the span in microseconds.
         end_time: End time of the span in microseconds.
         span_type: Type of the span. Can be a pre-defined enum or a custom string.
-        inputs: Input data of the span. Optional.
-        outputs: Outputs data of the span. Optional.
         attributes: Arbitrary key-value pairs of the span attributes. Optional.
         events: List of events that happened during the span. Optional.
     """
 
     name: str
     context: SpanContext
-    parent_span_id: Optional[str]
-    status: SpanStatus
+    parent_id: Optional[str]
+    status_code: str
+    status_message: Optional[str]
     start_time: int
     end_time: int
-    span_type: str = SpanType.UNKNOWN
-    inputs: Optional[Any] = None
-    outputs: Optional[Any] = None
```

Comment:
I think this is already planned in a follow-up PR, but can we make sure that we still expose `inputs` and `outputs` as convenience properties that Python users can access so they don't have to parse attributes themselves?

Reply:
Exactly! This class is unified with `MlflowSpanWrapper` which will expose accessor properties.